artiq/dashboard/datasets.py

```diff
@@ -83,16 +83,29 @@ def get_edit_widget_value(self):
 
 
 class Model(DictSyncTreeSepModel):
-    def __init__(self,  init):
-        DictSyncTreeSepModel.__init__(self, ".",
-                                      ["Dataset", "Persistent", "Value"],
-                                      init)
+    def __init__(self, init):
+        DictSyncTreeSepModel.__init__(
+            self, ".", ["Dataset", "Persistent", "HDF5 filters", "Value"], init
```

Comment:
I would opt for leaving out `HDF5 filters` here. That table is already very dense and hard to read because the `Persistent` column is a visual barrier that makes it hard to associate key and the corresponding value. Now with a potentially wide filter column in between this becomes even trickier.